portfolio/src/main/java/com/google/sps/servlets/DateServlet.java

```diff
@@ -0,0 +1,21 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.util.Date;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/date")
+public class DateServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    response.setContentType("text/html;");
+    response.getWriter().println("The server's current date is " + new Date());
+    
+  }
+
```

Comment:
Small nit: It's common to remove empty lines like this one at the end of java files. Commonly we add a line above a new method to separate it (like you did in line 13), but if it's the last method, we don't add a space! :)